ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -103,6 +103,10 @@ objects:
         value: /tmp/home
       - name: KUBECONFIG
         value: /tmp/artifacts/installer/auth/kubeconfig
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: BASE_DOMAIN
```

Comment:
setup should be the only container that consumes base_domain from env, other containers must get that info from it.

This is important because Azure is going to use different base domain, and this template is for all platforms.

Reply:
Also why not use something like https://github.com/openshift/installer/blob/f333d684f6553fcaf4f27e263794905da5abc6e2/data/data/bootstrap/files/usr/local/bin/installer-gather.sh#L104
To make this information gather process more appropriate?

Reply:
We need BASE_DOMAIN to update Route53 entries. These would be kept in sync as its being a template param

Reply:
Apparently its not required - https://github.com/openshift/release/pull/3842/files#r287372750